src/run_server.sh

```diff
@@ -99,8 +99,13 @@ function apply_preinstall_config() {
 function update_folder_permissions() {
     printf "\n### Updating Folder Permissions...\n"
 
-    chown -R "$(id -u):$(id -g)" "$BASE_GAME_DIR"
-    chown -R "$(id -u):$(id -g)" "$CONFIG_DIR"
+    if [ -z "$NO_CHOWN_GAME_DIR" ]; then
```

Comment:
Would you mind changing this to the "[[ ... ]]" notation?
It makes it more consistent with the rest of the script, and (as far as I know) it is the [recommended notation](https://google.github.io/styleguide/shellguide.html#s6.3-tests).